src/OidcClient/Infrastructure/LogSerializer.cs

```diff
@@ -1,9 +1,11 @@
 ﻿// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
 
-
 using System.Text.Json;
 using System.Text.Json.Serialization;
+#if NET5_0_OR_GREATER
```

Comment:
Why do we need this conditional compilation here, when it wasn't necessary in the other PR for IdentityModel itself?